src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -2070,6 +2091,10 @@ namespace OutputReportTabular {
                     displayEconomicResultSummary = true;
                     WriteTabularFiles = true;
                     nameFound = true;
+                } else if (UtilityRoutines::SameString(AlphArray(iReport), "HeatEmissionsSummary")) {
+                    displayHeatEmissionsSummary = true;
```

Comment:
Also add this line to the lists under AllSummary, AllSummaryAndMonthly, etc. [here](https://github.com/NREL/EnergyPlus/blob/bd2dd030748bb8199260801bc665f893a7d069e9/src/EnergyPlus/OutputReportTabular.cc#L2109) and following. Actually, that section could be consolidated with ||s to have just one list for summary, one list for monthly, etc. But that's not your job here. The monthly reports are done with a loop, so that should already be covered.